### PR TITLE
Sx42 user can logout

### PR DESCRIPTION
**Pull request by ahmad3m3**
@maggie-moheb: Kindly Review  the test, @ahmad-diaa : Kindly review the code, @daliamaarek : kindly review the Scenario, @azamzamy : kindly review the documentation.

### CODE REVIEW

**Comment by AmirMaurice** [comment]
@ahmad3m3 where's ur class diagram & wireframe ?? About the sequence diagram the 2 arrows in the same level specify which one is failure and which one is success and in the alt box specify what happens in case of both failure and success

**Comment by ahmad3m3** [comment]
@AmirMaurice : Reagarding the class diagram and wireframes you will find them in master, sequnce diagrams are modified, please recheck them.

**Comment by AmirMaurice** [comment]
@ahmad3m3 :+1:

**Comment by maggie-moheb** [comment]
@ahmad3m3 test is great :+1:

**Comment by daliamaarek** [comment]
Scenario works!

File: SmartX/app/src/main/java/com/smartx/cookies/smartx/ViewRooms.java [hunk 1]
```diff
@@ -293,6 +289,30 @@ public void failure(RetrofitError error) {
     }
 
     /**
+     * When the user click on logout, this method is called, it sends a request to delete the user's session after it succeed it renders login View after it sends
```
Comment (azamzamy) [inline SmartX/app/src/main/java/com/smartx/cookies/smartx/ViewRooms.java:310]: Could you check the spelling please?

File: SmartX/app/src/main/java/com/smartx/cookies/smartx/myAPI.java [hunk 2]
```diff
@@ -32,6 +34,9 @@
     @GET("/users/{userID}/rooms/1/devices/1/devices")
     void allDevices(@Path("userID") String id, Callback<List<Device>> callback);
```
Comment (azamzamy) [inline SmartX/app/src/main/java/com/smartx/cookies/smartx/myAPI.java]: could you document your method in myApi.java, please?

File: SmartX/app/src/main/java/com/smartx/cookies/smartx/myAPI.java [hunk 3]
```diff
@@ -32,6 +34,9 @@
...
 
+    @DELETE ("/session/{token}")
```
Comment (azamzamy) [inline SmartX/app/src/main/java/com/smartx/cookies/smartx/myAPI.java:39]: Could you document your method?

**Comment by ahmad3m3** [comment]
@azamzamy : kindly recheck the documentation.

**Pull request merged**